Tests/RegexBuilderTests/AlgorithmsTests.swift

```diff
@@ -104,4 +104,49 @@ class RegexConsumerTests: XCTestCase {
        result: "9+16, 3, 10, 99+1")
     )
   }
+
+  func testSwitches() {
+    // Failure cases
+    do {
+      switch "abcde" {
+      case Regex {
+        "a"
+        ZeroOrMore(.any)
+        "f"
+      }:
+        XCTFail()
+
+      case "abc":
+        XCTFail()
+
+      case Regex {
+        "a"
+        "b"
+        "c"
+      }:
+        XCTFail()
+
+      default:
+        return
```

Comment:
```suggestion
        break
```

Reply:
Otherwise the other tests in this function won't run. Also, would be nice to have a successful test case somewhere for switches.

Reply:
Fixed.